apps/api/src/app/events/e2e/send-message-push.e2e.ts

```diff
@@ -0,0 +1,207 @@
+import axios from 'axios';
+import { expect } from 'chai';
+import {
+  ExecutionDetailsRepository,
+  IntegrationRepository,
+  MessageRepository,
+  NotificationTemplateEntity,
+} from '@novu/dal';
+import { DetailEnum } from '@novu/application-generic';
+import { ChannelTypeEnum, PushProviderIdEnum, StepTypeEnum } from '@novu/shared';
+import { UserSession } from '@novu/testing';
+
+const axiosInstance = axios.create();
+
+describe('Trigger event - Send Push Notification - /v1/events/trigger (POST)', () => {
+  let session: UserSession;
+  let template: NotificationTemplateEntity;
+
+  const executionDetailsRepository = new ExecutionDetailsRepository();
+  const integrationRepository = new IntegrationRepository();
+  const messageRepository = new MessageRepository();
+
+  before(async () => {
+    process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = 'true';
+
+    session = new UserSession();
+    await session.initialize();
+
+    template = await session.createTemplate({
+      steps: [
+        {
+          active: true,
+          type: StepTypeEnum.PUSH,
+          title: 'Title',
+          content: 'Welcome to {{organizationName}}' as string,
+        },
+      ],
+    });
+  });
+
+  after(() => {
+    process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = 'false';
+  });
+
+  describe('Multiple providers active', () => {
+    before(async () => {
+      const payload = {
+        providerId: PushProviderIdEnum.EXPO,
+        channel: ChannelTypeEnum.PUSH,
+        credentials: { apiKey: '123' },
+        _environmentId: session.environment._id,
+        active: true,
+        check: false,
+      };
+
+      await session.testAgent.post('/v1/integrations').send(payload);
+
+      const integrations = await integrationRepository.find({
+        _environmentId: session.environment._id,
+        channel: ChannelTypeEnum.PUSH,
```

Comment:
Should we add active:true flag?

Reply:
It will protect this from failures. Good idea.